Optimal_bounding_box/include/CGAL/Optimal_bounding_box/nelder_mead_functions.h

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) 2018 GeometryFactory (France).
+// All rights reserved.
+//
+// This file is part of CGAL (www.cgal.org).
+// You can redistribute it and/or modify it under the terms of the GNU
+// General Public License as published by the Free Software Foundation,
+// either version 3 of the License, or (at your option) any later version.
+//
+// Licensees holding a valid commercial license may use this file in
+// accordance with the commercial license agreement provided with the software.
+//
+// This file is provided AS IS with NO WARRANTY OF ANY KIND, INCLUDING THE
+// WARRANTY OF DESIGN, MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE.
+//
+// $URL$
+// $Id$
+// SPDX-License-Identifier: GPL-3.0+
+//
+//
+// Author(s)     : Konstantinos Katrioplas
+
+#ifndef CGAL_OPTIMAL_BOUNDING_BOX_NEALDER_MEAD_FUNCTIONS_H
+#define CGAL_OPTIMAL_BOUNDING_BOX_NEALDER_MEAD_FUNCTIONS_H
+
+#include <CGAL/assertions.h>
+#include <vector>
+
+#if defined(CGAL_EIGEN3_ENABLED)
+#include <CGAL/Eigen_linear_algebra_traits.h>
+#endif
+
+
+
+namespace CGAL {
+namespace Optimal_bounding_box {
+
+#if defined(CGAL_EIGEN3_ENABLED)
+typedef CGAL::Eigen_linear_algebra_traits Linear_algebra_traits;
```

Comment:
You cannot simply place this `typedef` in the namespace.  @sloriot do you agree?